plutus-core/plutus-ir/test/transform/inline/letTypeAppMultiNotSat.golden

```diff
@@ -3,5 +3,5 @@
   (termbind
     (strict) (vardecl idFun (all a (type) (fun a a))) (abs a (type) (lam x a x))
   )
-  { idFun (con integer) }
+  [ (lam x (fun (con integer) (con integer)) x) (lam x (con integer) x) ]
```

Comment:
🆗

Reply:
It doesn't seem like this is bigger. The size of `{ idFun (fun (con integer) (con integer)) }` is 5, and so is the size of `(lam x (fun (con integer) (con integer)) x)`.

Reply:
Ah right, I wasn't looking at the _original_ term.